MIP/mip-2/README.md

```diff
@@ -0,0 +1,87 @@
+# MIP-02: Stage 0 Upgradeability and Multisigs
+- **Description**: Proposal to establish two multisigs, one for engineers and another for the foundation, to facilitate the deployment and upgrade of the Movement Token (MOVEToken) and other upgradeable contracts.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Desiderata**: [MD-0](../MD/md-0)
+
+## Abstract
+
+MIP-02 advocates for the creation of two multisigs: one managed by Movement Labs engineers to propose contract upgrades and another managed by the Movement Foundation to execute them. This proposal enables testing of multisig services while ensuring secure and transparent upgrade processes for the Movement Token. The multisigs will be deployed on Ethereum, Sepolia, and Holesky (if available) networks.
+
+## Motivation
+
+There is an immediate need to deploy the Movement Token (MOVEToken) using a robust, secure, and auditable upgrade mechanism. By creating two multisigs—one for proposal and one for execution—we ensure a transparent and decentralized governance process over upgrades to critical contracts.
+
+The first multisig allows Movement Labs engineers to propose upgrades via Safe’s multisig service. A remote key will automate the population of upgrade proposals. The second multisig, held by the Movement Foundation, will be responsible for executing the scheduled upgrades after a time delay, ensuring community trust and review.
+
+## Specification
+
+### Multisig Setup
+
+1. **Movement Labs Multisig** (4/5 signatures required):  
+   - Role: Proposing upgrades.
+   - Addresses: Deployed on Ethereum, Sepolia, and Holesky (if available).
+   - Signers: Andy Bell, Richard Melkonian, Liam Monninger, Primata (Engineers involved with Token), and a remote key for automated transaction proposals via SafeKit API's [proposeTransaction](https://docs.safe.global/sdk/api-kit#propose-a-transaction-to-the-service).
```

Comment:
I would list the account addresses here once we have them.

Manner of proposal is clear via the link. Good!

Reply:
Will add them as they come